source/release-notes/1.5.txt

```diff
@@ -0,0 +1,53 @@
+.. _c2c-release-notes-1.5:
+
+===================================
+Release Notes for mongosync 1.5
+===================================
+
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _1.5.0-c2c-release-notes:
+
+1.5.0 Release
+-------------
+
+**Upcoming**
+
+Oplog Rollover Resilience
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/fact-oplog-background
+
+Logging Level
+~~~~~~~~~~~~~
+
+Starting in 1.5, the default logging level is ``DEBUG``.  To change
+the logging level, see the :setting:`verbosity` setting.
+
+
+Bug Fixes
+---------
+
+- Fixes an issue where very large collections could timeout during 
+  ``mongosync`` initialization. 
+
+- Fixes an issue where ``mongosync`` could incorrectly report indexes 
+  as mismatched.
+
+- Fixes an issue where the :ref:`/start <c2c-api-start>` endpoint would not return
+  an error when passed a ``sharding`` object without the ``shardingEntries``
+  key.
+
+- Changes telemetry to send less metadata to Segment.
+
+Minimum Supported Version
+-------------------------
+
+In 1.5, the minimum supported version of MongoDB is 6.0.5.
```

Comment:
This should be 6.0.8 (for the server fixes for the data loss bug).

Reply:
Fixed